woocommerce/woocommerce-ios
# Update `InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView` design

Closes: #11229

## Description
This PR updates the design for `InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView` by updating both the text and the image that is being displayed. There are no logic changes involved, we just update the UI to clarify COD usage. Context: p1700477749579599-slack-C025A8VV728


## Testing instructions
1. On a store with IPP not completely setup ( you can use https://atomicippwcpaytests.wpcomstaging.com/ )
2. The easier way to force the view through the flow is [to comment out View's](https://github.com/woocommerce/woocommerce-ios/blob/c85cd6db8605455cc8900f2b06d4f2b447b1f63c/WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person%20Payments/InPersonPaymentsViewController.swift#L37-L109) `var body: some View` in `InPersonPaymentsViewController`, and use the specific View for this case instead:
```
  InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpView(
    viewModel: InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpViewModel(
    plugin: .wcPay,
    analyticReason: viewModel.state.reasonForAnalytics,
    completion: viewModel.refresh))
```
3. Run the app > Menu > Payments > See the `In-Person Payments setup is incomplete. Continue Setup` message at the bottom > Tap the message
4. Observe the updated design like the screenshots below.

## Screenshots
| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 - 2023-12-06 at 15 03 49](https://github.com/woocommerce/woocommerce-ios/assets/3812076/5cf4e108-8100-4308-bf73-fb43280e56fc) | ![Simulator Screenshot - iPhone 15 - 2023-12-06 at 15 33 14](https://github.com/woocommerce/woocommerce-ios/assets/3812076/5fd6780a-650d-40ae-986e-39d00ed1f45d)|
